.github/workflows/ci.yml

```diff
@@ -54,3 +52,24 @@ jobs:
 
       - name: Run tests
         run: npm test --ignore-scripts
+
+  smoketest:
+    runs-on: ubuntu-latest
+    # if: ${{ github.actor != 'dependabot[bot]' }}
+    strategy:
+      matrix:
+        regex:
```

Comment:
If I'm reading the docs correctly, this will run 4 different jobs in parallel, is that correct?

If that's right, then you actually should remove jobs 2 and 4, and just keep 1 and 3. Job 2 would run through _everything_ except `^az` and job 4 would run through _everything_ except `^go`. I think 1 and 3 alone would cover everything you want to cover.

(Also, if you want to speed things up, maybe spit the B-Z job into smaller chunks, so that more happens in parallel?)

Reply:
I'm actually refactoring this to split it up into 15 different chunks based on the total number of specs in the dataset. Doing this regex thing is really messy.